src/System.Diagnostics.Debug/ref/System.Diagnostics.Debug.cs

```diff
@@ -8,105 +8,461 @@
 
 namespace System.Diagnostics
 {
+    /// <summary>
+    /// Provides a set of methods and properties that help debug your code. This class cannot be inherited.
```

Comment:
Ok, my bad. You're right @cartermp. We do say that for most of the sealed classes (I say most because this is not automatically added so the writer has to remember to add that). I don't think we do this for static classes. I'll get that fixed for the Debug class,